### ISSUE TITLE
No refresh worker logs when running Cube in production mode

### ISSUE THREAD

**Issue by anshjain18** [gadsme/charts#42]
Hi team,
Recently I switched my Cube setup to production mode, and noticed that my pre-aggregations were not getting updated as expected.
There were no logs whatsoever on either the API instance pod or refresh worker pod. As soon as I switched back to dev mode, logs were coming from refresh worker pod.
Is this the expected behaviour, because it would be quite difficult then to debug issues while running Cube in production mode.

This is my values.yaml for Cube
```
cube:
  image:
    repository: cubejs/cube
    tag: v0.34.40
    pullPolicy: IfNotPresent

  datasources:
    default: 
      type: snowflake
      user: CUBE_DEV_USER
      pass: qj4mp0IQ6Jvs
      schema: PUBLIC
      name: UC_DATA_DEV
      snowFlake:
        account: ft39330.ap-southeast-1
        region: ap-southeast-1
        role: DP_DW_USER_DEV_ROLE
        warehouse: CUBE_POC_WH
  
  cubestore:
    host: cubestore-dev-router.cubestore
    port: 3030

  config:
    preAggregationsSchema: pre_agg_schema
    debug: true
    logLevel: "debug"
    devMode: false
    rollupOnly: true
    pgSqlPort: 15432
    sqlUser: admin
    sqlPassword: admin
    volumes:
      - name: cube-schema
        configMap:
          name: cube-schema
      - name: cube-config
        configMap: 
          name: cube-config
```
1. debug is set to true
2. logLevel is set to "debug"

@lvauvillier

**Comment by lvauvillier** [comment]
Hello @anshjain18, we're using a similar configuration and everything is working fine for us.

Could you try adding cacheAndQueueDriver: cubestore to your configuration section? In development mode, the default setting is memory. This change might generate logs that could help identify the error.

**Comment by anshjain18** [comment]
Hi @lvauvillier, thanks would add this.
Though I can see that the default value for this setting is 'cubestore' only.

**Comment by lvauvillier** [comment]
Yes, the comments in the chart above the value fields may become outdated, as the Cube documentation is frequently updated.
The best is to refer to the official documentation. Most of the environment variables has a direct mapping in the chart values
For example, regarding `cacheAndQueueDriver`, you can refer to:
https://cube.dev/docs/reference/configuration/environment-variables#cubejs_cache_and_queue_driver
